DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/README.md

```diff
@@ -207,10 +209,10 @@ The `Board Test` program checks following interfaces in a platform:
 
 - **Kernel-to-device global memory interface:** This interface is checked by performing kernel to memory data transfers using simple read and write kernels. Kernel to memory bandwidth is measured and reported.
 
-  > **Note**: This test currently does not support SYCL Unified Shared Memory (USM). For testing the USM interface, use the [Simple host streaming sample](/DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/simple_host_streaming) code sample in the oneAPI-sample GitHub repository.
-
 - **Host-to-kernel interface:** The test ensures that the host to kernel communication is correct and that the host can launch a kernel successfully. It also measures the roundtrip kernel launch latency and throughput (number of kernels/ms) of single task no-operation kernels.

```

Comment:
this should come first

Reply:
What are you referring to here? Sorry the GitHub UI here is a little confusing.